src/main/java/it/gov/pagopa/payhub/auth/service/AuditLoggerServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.payhub.auth.service;
+
+import it.gov.pagopa.payhub.auth.dto.AuditLogDTO;
+import it.gov.pagopa.payhub.auth.enums.AuditEventType;
+import it.gov.pagopa.payhub.auth.utils.AuditUtils;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j(topic = "KAFKA_LOGGER")
```

Comment:
let's keep it unaware of the destination, let's just declare the AUDIT logger
```suggestion
@Slf4j(topic = "AUDIT")
```